From 198ecc35d7bf9b20e4429ca0fc051a8126590ab4 Mon Sep 17 00:00:00 2001
From: Santiago Rossi <santiago_rho_psi@hotmail.com>
Date: Mon, 16 Apr 2012 21:21:25 -0300
Subject: [PATCH 23/40] 4.4.4 Cuenca de Sholazar: Serfex el Atracador

Converted "Serfex the Reaver" from EAI to SAI. Related to quest "The Underground Menace"
---
 ...2012_04_16_105_world_the_underground_menace.sql |   16 ++++++++++++++++
 1 files changed, 16 insertions(+), 0 deletions(-)
 create mode 100644 sql/updates/world/reanemu/2012_04_16_105_world_the_underground_menace.sql

diff --git a/sql/updates/world/reanemu/2012_04_16_105_world_the_underground_menace.sql b/sql/updates/world/reanemu/2012_04_16_105_world_the_underground_menace.sql
new file mode 100644
index 0000000..1787ec9
--- /dev/null
+++ b/sql/updates/world/reanemu/2012_04_16_105_world_the_underground_menace.sql
@@ -0,0 +1,16 @@
+-- Serfex the Reaver SAI
+SET @SERFEX := 28083;
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`= 28083;
+DELETE FROM `creature_ai_scripts` WHERE `creature_id`=@SERFEX;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=@SERFEX AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
+(@SERFEX,0,0,1,25,0,100,0,0,0,0,0,11,20567,0,0,0,0,0,1,0,0,0,0,0,0,0,'Serfex the Reaver - Reset - Cast Submerge Visual'),
+(@SERFEX,0,1,2,61,0,100,0,0,0,0,0,11,29147,0,0,0,0,0,1,0,0,0,0,0,0,0,'Serfex the Reaver - Reset - Cast Tunnel Bore'),
+(@SERFEX,0,2,3,61,0,100,0,0,0,0,0,18,33554434,0,0,0,0,0,1,0,0,0,0,0,0,0,'Serfex the Reaver - Reset - Set Unit Flags'),
+(@SERFEX,0,3,0,61,0,100,0,0,0,0,0,21,1,0,0,0,0,0,1,0,0,0,0,0,0,0,'Serfex the Reaver - Reset - Allow combat movement'),
+(@SERFEX,0,4,5,4,0,100,0,0,0,0,0,28,20567,0,0,0,0,0,1,0,0,0,0,0,0,0,'Serfex the Reaver - Aggro - Remove aura Submerge Visual'),
+(@SERFEX,0,5,6,61,0,100,0,0,0,0,0,28,29147,0,0,0,0,0,1,0,0,0,0,0,0,0,'Serfex the Reaver - Aggro - Remove aura Tunnel Bore'),
+(@SERFEX,0,6,7,61,0,100,0,0,0,0,0,19,33554434,0,0,0,0,0,1,0,0,0,0,0,0,0,'Serfex the Reaver - Aggro - Remove Unit Flags'),
+(@SERFEX,0,7,8,61,0,100,0,0,0,0,0,21,0,0,0,0,0,0,1,0,0,0,0,0,0,0,'Serfex the Reaver - Aggro - Disable combat movement'),
+(@SERFEX,0,8,0,61,0,100,0,0,0,0,0,11,37752,0,0,0,0,0,1,0,0,0,0,0,0,0,'Serfex the Reaver - Aggro - Cast Stand'),
+(@SERFEX,0,9,0,0,0,100,0,1000,1000,2100,4500,11,31747,0,0,0,0,0,2,0,0,0,0,0,0,0,'Serfex the Reaver - Aggro - Cast Stand');
\ No newline at end of file
-- 
1.7.9.msysgit.0

